AnnotatoBackend/Sources/App/WebSocketControllers/WebSocketController.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+import Vapor
+import AnnotatoSharedLibrary
+import FluentKit
+
+class WebSocketController {
+    /// Each user has one websocket connection.
+    private static var openConnections: [String: WebSocket] = [:]
+    private static var logger = Logger(label: "WebSocketController")
+
+    static func handleIncomingConnection(req: Request, webSocket: WebSocket) {
+        guard let userId = try? ControllersUtil.getIdFromParamsAsString(request: req) else {
```

Comment:
After changing the route param to `userId`, I think it would be better to use `ControllersUtil.getParamValue` here.